src/main/cpp/Robot.cpp

```diff
@@ -17,9 +17,7 @@ void Robot::RobotInit() {}
  * <p> This runs after the mode specific periodic functions, but before
  * LiveWindow and SmartDashboard integrated updating.
  */
-void Robot::RobotPeriodic() {
-  frc2::CommandScheduler::GetInstance().Run();
-}
+void Robot::RobotPeriodic() { frc2::CommandScheduler::GetInstance().Run(); }
```

Comment:
I don't know how I feel about having the closing curly brace not be on its own line. Like I know it's minor but for the purposes of creating a style

Reply:
This is a clang-format auto-format thing. I don't like it either, I'll see if I can fix it in my next PR